src/OrbitGl/MultivariateTimeSeries.h

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_MULTIVARIATE_TIME_SERIES_H_
+#define ORBIT_GL_MULTIVARIATE_TIME_SERIES_H_
+
+#include <array>
+#include <map>
+#include <optional>
+#include <string>
+
+#include "OrbitBase/Logging.h"
+
+template <size_t Dimension>
+class MultivariateTimeSeries {
+  static_assert(Dimension >= 1, "Dimension must be at least 1");
+
+ public:
+  explicit MultivariateTimeSeries(std::array<std::string, Dimension> series_names)
+      : series_names_(std::move(series_names)) {}
+
+  [[nodiscard]] const std::array<std::string, Dimension>& series_names() const {
+    return series_names_;
+  }
+  [[nodiscard]] const std::map<uint64_t, std::array<double, Dimension>>& time_to_series_values()
+      const {
+    return time_to_series_values_;
+  }
+  [[nodiscard]] double min_series_value() const { return min_; }
+  [[nodiscard]] double max_series_value() const { return max_; }
+  [[nodiscard]] std::optional<uint8_t> value_decimal_digits() const {
+    return value_decimal_digits_;
+  }
+  [[nodiscard]] std::string value_unit() const { return value_unit_; }
+
+  void AddValues(uint64_t timestamp_ns, const std::array<double, Dimension>& values) {
+    time_to_series_values_[timestamp_ns] = values;
+    for (double value : values) {
+      UpdateMinAndMax(value);
+    }
+  }
+  void SetValueUnit(std::string value_unit) { value_unit_ = std::move(value_unit); }
```

Comment:
Nit: `set_value_unit()`. 

I would be consistent in this file. Aka: Either all methods MixedCase or accessors _and_ mutators like the variables.

Reply:
Changed to keep using the MixedCase for all methods. Thanks!